src/Components/Cart.js

```diff
@@ -0,0 +1,61 @@
+import bonusItems from "../data/bonusItems";
+export default function Cart({ cart, setCart}) {
+  let cartTotal = cart.reduce((a, b) => {
+    return b.amount + a;
+  }, 0);
+
+  //removing bird from cart
+  const updateBirdsArray = (bird) => {
+    cart.splice(cart.indexOf(bird), 1);
+
+    setCart([...cart]);
+  };
```

Comment:
A couple of notes about this component. I'll start by saying that you are passing the specs but the functionality isn't actually there. I tried to add and delete birds from your cart but ran into some weird bugs. You could solve this if when you map on lines 42-49, instead of passing a birdie to this function you passed the index of the bird in the array, and then you only need to use `splice` with that index. Right now your code doesn't delets the bird you clicked on, but the first occurrence of that bird in the array (indexOf finds the index of the first matching element in the array). If you use the index from when you mapped over the array you grab exactly the bird that the user clicked on. Let me know if you need further explanation about this, it's important.